sonar/config_sonar.py

```diff
@@ -17,6 +17,8 @@
 
 """Specific configuration SONAR."""
 
+SONAR_APP_BASE_URL = 'https://localhost:5000/'
```

Comment:
ARK use JSONSCHEMAHOST  for this: https://github.com/rero/sonar/blob/a9d5dc2ac477526c6cf03e395d268f1b12534389/sonar/modules/ark/api.py#L140